server/spec/api/v1/stacks_spec.rb

```diff
@@ -151,6 +151,16 @@
         expect(json_response['parent']).to match hash_including('name' => stack.name, 'id' => "#{stack.grid.name}/#{stack.name}")
       end
 
+      context 'when parent stack does not exist' do
+        it 'returns stack json with parent name excluding parent id' do
+          child_stack_1.destroy
+          get "/v1/stacks/#{child_stack_1_2.to_path}", nil, request_headers
+          expect(json_response['parent'].key?('name')).to be_truthy
+          expect(json_response['parent'].key?('id')).to be_falsey
+          expect(json_response['parent']).to match hash_including('name' => stack.name)
```

Comment:
I suggest the more explicit `expect(json_response['parent']).to eq { 'id' => nil, 'name' => ...}` without the `hash_including`.

Reply:
Using the `hash_including` is future proofier.

In case the future adds some other fields to the parent object, the spec would fail for no good reason.